build_tools/cmake/iree_installed_test.cmake

```diff
@@ -36,20 +36,39 @@ function(iree_add_installed_test)
     ${ARGN}
   )
 
+
   add_test(
-    NAME ${_RULE_TEST_NAME}
-    WORKING_DIRECTORY ${_RULE_WORKING_DIRECTORY}
+    NAME
+      ${_RULE_TEST_NAME}
     COMMAND
-        ${_RULE_COMMAND}
-    WORKING_DIRECTORY
-        "${CMAKE_BINARY_DIR}"
+      ${_RULE_COMMAND}
+  )
+  if (DEFINED _RULE_WORKING_DIRECTORY)
+    set_property(
+      TEST
+        ${_RULE_TEST_NAME}
+      PROPERTY WORKING_DIRECTORY
+        "${_RULE_WORKING_DIRECTORY}"
+    )
+  endif()
+  set_property(
+    TEST
+      ${_RULE_TEST_NAME}
+      PROPERTY LABELS
+        "${_RULE_LABELS}"
```

Comment:
nit: indentation?

```suggestion
  set_property(
    TEST
        ${_RULE_TEST_NAME}
      PROPERTY LABELS
        "${_RULE_LABELS}"
```

Reply:
I actually unindented the stuff below, but yea. I want an autoformatter :-/